src/Benchmarks.ServerJob/ServerJob.cs

```diff
@@ -62,5 +62,6 @@ public class ServerJob : IIdentifiable
         public WebHost WebHost { get; set; }
 
         public bool UseRuntimeStore { get; set; }
+        public TimeSpan Timeout { get; set; } = TimeSpan.FromMinutes(5);
```

Comment:
Should this be 2 minutes instead of 5?  Everything else says the default is 2 minutes.